controllers/infinispan_controller.go

```diff
@@ -300,21 +300,26 @@ func (reconciler *InfinispanReconciler) Reconcile(ctx context.Context, ctrlReque
 
 	// If an overlay configuration is specified wait for the related ConfigMap
 	overlayConfigMap := &corev1.ConfigMap{}
-	var overlayConfigMapKey string
+	var overlayConfigMapKey, overlayLog4jConfigMapKey string
```

Comment:
There's no need for the `overlayLog4jConfigMapKey` here as the filename is always `log4j.xml`. The reason we require `overlayConfigMapKey` is because we support three different names for the server configuration.